cdisc_rules_engine/services/data_services/local_data_service.py

```diff
@@ -89,6 +89,7 @@ def get_raw_dataset_metadata(self, dataset_name: str, **kwargs) -> DatasetMetada
             label=contents_metadata["dataset_label"],
             modification_date=contents_metadata["dataset_modification_date"],
             filename=file_metadata["name"],
+            full_path=file_metadata["path"],
```

Comment:
Why does the metadata path need to be different from the file path?

Reply:
Metadata path is the same as the file metadata, maybe I misunderstood your question?

Reply:
My question is why do we need a filename variable and a full_path variable? Shouldn't they be the same, or at least shouldn't we be able to get the filename from the full path